packages/blade/src/components/TopNav/_decisions/decisions.md

```diff
@@ -264,6 +473,298 @@ const Dashboard = () => {
   - The navigation bar will have `role=navigation` to indicate that it is a navigation landmark.
```

Comment:
can we have an announcement for screen readers when a TabNav item is clicked?

Reply:
We will have proper announcements on focus. 

Clicking generally doesn't require announcements, because on route change the router itself will announce any page state changes.